docs/architecture.md

```diff
@@ -0,0 +1,27 @@
+# onos-gui Architecture
+The GUI is architected as a Cloud Native application to be run on Kubernetes
+accessing the back end modules through gRPC (and in the case of the Config GUI
+additionally through gNMI)
+
+The front end technology is [Angular](https://angular.io) and [gRPC Web](https://github.com/grpc/grpc-web)
+is used to access the back end systems. [Nginx](https://nginx.org/) acts as a
+Proxy, Envoy acts as a grpc-web proxy and all are deployed on [Docker](https://www.docker.com/community/open-source).
```

Comment:
Is there a linkable page for Envoy we can put here?

Reply:
done